data/Cowling_1987/metadata.yml

```diff
@@ -0,0 +1,224 @@
+    to promote seed release (Cowling & Lamont 1985). Seeds were classified as firm
+    (apparently viable), aborted or predated. We tested the viability of firm seed
+    by germinating three replicate batches of fifteen seeds of each species in each
+    of three age classes (1-3, 4-6, 7-9 years), as well as 10-12 year-old seed of
+    B. leptophylla, in Petri dishes on moistened filter paper at 15 deg C over fifty
+    days.
+- var_in: seed_dry_mass_mg
+  unit_in: mg
+  trait_name: seed_dry_mass
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 50
+  methods: Seed masses do not include wings. We harvested all the cones from fifteen
+    randomly chosen individuals of each species. Cone age was estimated from node
+    counts except for B. leptophylla where cones were grouped into four classes (1-3,
+    4-6, 7-9, 10-12 years) on the basis of the relative persistence and extent of
+    fading of the floral remnants on the cone. Cones were returned to the laboratory
+    where the numbers of open, closed and predated follicles per cone were counted
+    (Scott 1982). We burned the cones with an oxy-acetyline torch until all follicles
+    had ruptured. The burnt cones were placed in trays in a glasshouse and periodically
+    immersed in water to promote seed release (Cowling & Lamont 1985). Seeds were
+    classified as firm (apparently viable), aborted or predated. We tested the viability
+    of firm seed by germinating three replicate batches of fifteen seeds of each species
+    in each of three age classes (1-3, 4-6, 7-9 years), as well as 10-12 year-old
+    seed of B. leptophylla, in Petri dishes on moistened filter paper at 15 deg C
+    over fifty days.
+- var_in: seed_length_min_mm
+  unit_in: mm
+  trait_name: seed_length
+  entity_type: species
+  value_type: minimum
+  basis_of_value: measurement
+  replicates: unknown
+  methods: From literature (George 1981) and this study.
+- var_in: seed_length_max_mm
+  unit_in: mm
+  trait_name: seed_length
+  entity_type: species
+  value_type: maximum
+  basis_of_value: measurement
+  replicates: unknown
+  methods: From literature (George 1981) and this study.
+- var_in: seed_width_min_mm
+  unit_in: mm
+  trait_name: seed_width
+  entity_type: species
+  value_type: minimum
+  basis_of_value: measurement
+  replicates: unknown
+  methods: From literature (George 1981) and this study.
+- var_in: seed_width_max_mm
+  unit_in: mm
+  trait_name: seed_width
+  entity_type: species
+  value_type: maximum
+  basis_of_value: measurement
+  replicates: unknown
+  methods: From literature (George 1981) and this study.
+- var_in: reproductive_maturity_min
+  unit_in: a
+  trait_name: fire_time_from_fire_to_flowering
+  entity_type: population
+  value_type: minimum
+  basis_of_value: measurement
+  replicates: unknown
+  methods: We harvested all the cones form fifteen randomly chosen individuals of
```

Comment:
I left it as unknown because I assumed they'd have looked at all individuals of that age in the population to decide if flowering had commenced. But it could also be `.na` - maybe that is better because it indicates I've thought about it.

Reply:
I've gone back and forth on this several times and now have put in 15. I assumed they'd based some information on all individuals in the population, but thinking more about counting stem nodes, it does imply they only looked at these 15 plants.